contracts/adapters/Hyperlane/HyperlaneSenderAdapter.sol

```diff
@@ -86,14 +86,22 @@ contract HyperlaneSenderAdapter is IBridgeSenderAdapter, BaseSenderAdapter, Owna
         // See https://docs.hyperlane.xyz/docs/build-with-hyperlane/guides/paying-for-interchain-gas
         if (address(igp) != address(0) && dstDomainId != 0) {
             // destination gasAmount is hardcoded to 500k similar to Wormhole implementation
-            return igp.quoteGasPayment(dstDomainId, 500000);
+            try igp.quoteGasPayment(dstDomainId, 500000) returns (uint256 gasQuote) {
```

Comment:
Nit: Can we just remove the if clause in that case? Also wonder if we should do that in the actual dispatch below

Reply:
updated